pypesto/hierarchical/relative/problem.py

```diff
@@ -66,43 +65,35 @@ def from_petab_amici(
             petab_problem, amici_model, edatas
         )
 
-    def check_edatas(self, edatas: list[amici.ExpData]) -> bool:
-        """Check for consistency in data.
-
-        Currently only checks for the actual data values. e.g., timepoints are
-        not compared.
-
-        Parameters
-        ----------
-        edatas:
-            A data set. Will be checked against the data set provided to the
-            constructor.
-
-        Returns
-        -------
-        Whether the data sets are consistent.
-        """
-        # TODO replace but edata1==edata2 once this makes it into amici
-        #  https://github.com/AMICI-dev/AMICI/issues/1880
-        data = [
-            amici.numpy.ExpDataView(edata)["observedData"] for edata in edatas
-        ]
-
-        if len(self.data) != len(data):
-            return False
+    def get_relative_observable_ids(self) -> list[str]:
+        """Get unique list of IDs of all relative observables with scaling and/or offset."""
+        return list(
+            {
+                x.observable_id
+                for x in self.xs.values()
+                if x.inner_parameter_type
+                in [
+                    InnerParameterType.SCALING,
+                    InnerParameterType.OFFSET,
+                ]
+            }
+        )
 
-        for data0, data1 in zip(self.data, data):
-            if not np.array_equal(data0, data1, equal_nan=True):
-                return False
+    def get_groups_for_xs(self, inner_parameter_type: str) -> list[int]:
+        """Get unique list of ``RelativeParameter.group`` values."""
+        groups = [x.group for x in self.get_xs_for_type(inner_parameter_type)]
+        return list(set(groups))
 
-        return True
+    def get_xs_for_group(self, group: int) -> list[RelativeInnerParameter]:
+        r"""Get ``RelativeParameter``\s that belong to the given group."""
```

Comment:
Does this fail?

```suggestion
        """Get ``RelativeParameter``s that belong to the given group."""
```

Reply:
No need, but I guess the `\s` is problematic?

Reply:
Yeah, interestingly the docs test fails with
```
Warning, treated as error:
/home/runner/work/pyPESTO/pyPESTO/pypesto/hierarchical/relative/problem.py:docstring of pypesto.hierarchical.relative.problem.RelativeInnerProblem.get_xs_for_obs_idx:1:Inline literal start-string without end-string.
doc: exit 2 (611.35 seconds) /home/runner/work/pyPESTO/pyPESTO> sphinx-build -W -b html doc/ doc/_build/html pid=2938
  doc: FAIL code 2 (617.84=setup[6.49]+cmd[611.35] seconds)
  evaluation failed :( (618.03 seconds)
Error: Process completed with exit code 2.
```

Reply:
Ah OK. These docs suggest `\ s`, not `\s`, but if you found something that works then nevermind!

https://documatt.com/restructuredtext-reference/element/inline-literal.html#requires-whitespace-or-punctuation-around

Reply:
Ohh that's the issue?
Ok, will leave it as is for now, but thanks for checking. Good to have in mind.